core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -159,23 +166,35 @@
    private static final byte RESPONSE = 1;
    private static final byte SINGLE_MESSAGE = 2;
 
-   @Inject protected GlobalConfiguration configuration;
-   @Inject @ComponentName(KnownComponentNames.INTERNAL_MARSHALLER)
+   @Inject
+   protected GlobalConfiguration configuration;
+   @Inject
+   @ComponentName(KnownComponentNames.INTERNAL_MARSHALLER)
    protected StreamingMarshaller marshaller;
-   @Inject protected CacheManagerNotifier notifier;
-   @Inject protected TimeService timeService;
-   @Inject protected InboundInvocationHandler invocationHandler;
-   @Inject @ComponentName(KnownComponentNames.TIMEOUT_SCHEDULE_EXECUTOR)
+   @Inject
+   protected CacheManagerNotifier notifier;
+   @Inject
+   protected TimeService timeService;
+   @Inject
+   protected InboundInvocationHandler invocationHandler;
+   @Inject
+   @ComponentName(KnownComponentNames.TIMEOUT_SCHEDULE_EXECUTOR)
    protected ScheduledExecutorService timeoutExecutor;
-   @Inject @ComponentName(KnownComponentNames.NON_BLOCKING_EXECUTOR)
+   @Inject
+   @ComponentName(KnownComponentNames.NON_BLOCKING_EXECUTOR)
    protected ExecutorService nonBlockingExecutor;
-   @Inject protected CacheManagerJmxRegistration jmxRegistration;
-   @Inject protected GlobalXSiteAdminOperations globalXSiteAdminOperations;
+   @Inject
+   protected CacheManagerJmxRegistration jmxRegistration;
+   @Inject
+   protected GlobalXSiteAdminOperations globalXSiteAdminOperations;
```

Comment:
Would be great if we could prevent the reformatting here as it is creating quite a bit of additional noise.

Reply:
Cleaned it up.